lib/core/src/irods_configuration_keywords.cpp

```diff
@@ -9,16 +9,17 @@ namespace irods
     const char* const KW_CFG_PAM_PASSWORD_MAX_TIME{"password_max_time"};
     const char* const KW_CFG_PAM_PASSWORD_EXTEND_LIFETIME{"password_extend_lifetime"};
 
-    const char* const KW_CFG_DB_HOST{"db_host"};
-    const char* const KW_CFG_DB_PORT{"db_port"};
-    const char* const KW_CFG_DB_NAME{"db_name"};
-    const char* const KW_CFG_DB_ODBC_DRIVER{"db_odbc_driver"};
-    const char* const KW_CFG_DB_USERNAME{"db_username"};
-    const char* const KW_CFG_DB_PASSWORD{"db_password"};
-    const char* const KW_CFG_DB_SSLMODE{"db_sslmode"};
-    const char* const KW_CFG_DB_SSLROOTCERT{"db_sslrootcert"};
-    const char* const KW_CFG_DB_SSLCERT{"db_sslcert"};
-    const char* const KW_CFG_DB_SSLKEY{"db_sslkey"};
+    const char* const KW_CFG_DB_TECHNOLOGY{"technology"};
+    const char* const KW_CFG_DB_HOST{"host"};
+    const char* const KW_CFG_DB_PORT{"port"};
+    const char* const KW_CFG_DB_NAME{"name"};
+    const char* const KW_CFG_DB_ODBC_DRIVER{"odbc_driver"};
+    const char* const KW_CFG_DB_USERNAME{"username"};
+    const char* const KW_CFG_DB_PASSWORD{"password"};
+    const char* const KW_CFG_DB_SSLMODE{"sslmode"};
+    const char* const KW_CFG_DB_SSLROOTCERT{"sslrootcert"};
+    const char* const KW_CFG_DB_SSLCERT{"sslcert"};
+    const char* const KW_CFG_DB_SSLKEY{"sslkey"};
```

Comment:
Save as a final step: `ssl` -> `tls`

Reply:
Just to clarify, all `sll*` in the database plugin config should be `tls*` now?

Reply:
Yes.

Reply:
Unless there's a strong reason not to do that. See where those options are used before renaming them.

Reply:
Should we update the `KW_*SSL*` to `KW_*TLS*` too?

Reply:
If there's only a few instances, yes.

Reply:
Done.